Trippy/Handlers/CacheDataHandler.m

```diff
@@ -150,6 +150,10 @@ - (void) fetchSavedItineraries {
                 [strongSelf.delegate generalRequestFail:error];
             } else {
                 strongSelf.itineraryFetchCount = objects.count;
+                if (objects.count == 0) {
+                    __strong CacheDataHandler *strongSelf = weakSelf;
```

Comment:
since you have already added it in line 148, you shouldn't need to redo it? (same as below)